lib/internal/util/inspect.js

```diff
@@ -1285,8 +1285,14 @@ function identicalSequenceRange(a, b) {
   return { len: 0, offset: 0 };
 }
 
-function getStackString(error) {
-  return error.stack ? String(error.stack) : ErrorPrototypeToString(error);
+function getStackString(ctx, error) {
+  if (error.stack) {
+    if (ArrayIsArray(error.stack)) {
+      return formatArray(ctx, error.stack);
+    }
+    return String(error.stack);
```

Comment:
If we want to really be safe, we can just use `formatValue()` for everything that's not a string.

```suggestion
    if (typeof error.stack === 'string) {
      return error.stack;
    }
    return formatValue(ctx, error.stack);
```

Reply:
This will change existing output - the `[[1, 2, 3], '1,2,3: foo', '[1,2,3]']` case for example, which currently does not obey indentation.

Reply:
I believe that's an improvement to the output, so I would not mind that :)